forwarding/forwarding.go

```diff
@@ -4,16 +4,16 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/Portshift/klar/clair"
 	"github.com/Portshift/klar/docker"
 	"github.com/Portshift/klar/types"
+	grype_models "github.com/anchore/grype/grype/presenter/models"
 	log "github.com/sirupsen/logrus"
 	"io/ioutil"
 	"net/http"
 )
 
 type ImageVulnerabilities struct {
-	Vulnerabilities []*clair.Vulnerability   `json:"vulnerability"`
+	Vulnerabilities *grype_models.Document   `json:"vulnerability"`
```

Comment:
why is the json annotaion is vulnerability and not vulnerabilities ?

Reply:
didn't touch it, don't want to break stuff that might depend on it across the flow